documentation/properties/incurred_cva.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout:		property
 title:		"incurred_cva"
-schemas:	[agreement]
+schemas:	[entity]
```

Comment:
This should be specific to the customer schema only as I don't think it would make sense for guarantor/issuer to have incurred_cva charges?

Reply:
absolutely. I did not know there was a customer schema. thank you!